rudder-agent/SOURCES/debug-script/tests/rudder_unsent_inventories_agent

```diff
@@ -20,11 +20,19 @@ rudder_unsent_inventories_agent() {
   else
     UNSENT_INVENTORIES_DIR="${RUDDER_VAR}/inventories"
   fi
-
-  COMMAND_OUTPUT=`ls -l "${UNSENT_INVENTORIES_DIR}" 2>&1 | wc -l`
+  recent_time="432000"
+  for i in "${UNSENT_INVENTORIES_DIR}"
+  do
+    last_modif=$(stat -c %Y $i)
+    recent_modif=$(($last_modif-$recent_time))
+    if [ $last_modif -ge $recent_modif ]
+    then
+      COMMAND_OUTPUT=`ls -l "${UNSENT_INVENTORIES_DIR}" 2>&1 | wc -l`
```

Comment:
You still give the full directory output so there is no difference with what was done before